ValidationLibrary.AzureFunctions.Tests/StatusEndpointTests.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using NUnit.Framework;
+using Octokit;
+using ValidationLibrary.Rules;
+
+namespace ValidationLibrary.AzureFunctions.Tests
+{
+    [TestFixture]
+    public class StatusEndpointTests
+    {
+        private StatusEndpoint _statusEndpoint;
+
+        [Test]
+        public void Run_NormalStatusEndpointCheck()
+        {
+            Environment.SetEnvironmentVariable("GitHub:Organization", "mock");
+            Environment.SetEnvironmentVariable("GitHub:Token", "mock");
```

Comment:
Remove this, we don't need env variables when we are testing `StatusEndpoint`

Reply:
Oh yes forget to remove with the host builder